assets/src/design-system/components/menu/useDropDownMenu.js

```diff
@@ -46,15 +47,34 @@ export default function useDropDownMenu({
   const listLength = allOptions.length;
   const [focusedValue, setFocusedValue] = useState(activeValue);
 
+  const getFocusedIndex = useCallback(
+    () =>
+      allOptions.findIndex(
+        (option) => option?.value?.toString() === focusedValue.toString()
+      ),
+    [allOptions, focusedValue]
+  );
+
+  // there's an edge case in this menu when the activeValue given to a menu isn't present in the options passed to it that we want to check against when this first renders.
```

Comment:
💭 break this over two lines so it's more readable on smaller screens/bigger fonts?
```suggestion
  // there's an edge case in this menu when the activeValue given to a menu
  // isn't present in the options passed to it that we want to check against when this first renders.
```